Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rentpath/rp_node:8.9.1
+
+WORKDIR $HOME/react-ui
+COPY --chown=node:node package.json yarn.lock $HOME/react-ui/
+run chown -R node:node $HOME/react-ui
+
+USER root
+COPY --chown=node:node . $HOME/react-ui/
+RUN apt-get install -y nginx
+
+USER node
+RUN yarn config set registry https://registry.yarnpkg.com
+RUN yarn install
+
+ARG VERSION
+ARG COMMIT
+ARG BUILD_NUMBER
```

Comment:
Are you actually using these build args?

Reply:
We probably don't need these at all actually.

Reply:
No, that was an area I thought we might talk about. Where does that stuff come in and do we need it for this type of repo?

Reply:
Removed